docker-compose/docker-compose.yaml

```diff
@@ -2,23 +2,57 @@ version: '2.4'
 services:
   # Description: Acts as a reverse proxy for all of the sourcegraph-frontend instances
   #
-  # Disk: none
+  # Disk: 1GB / persistent SSD
   # Ports exposed to other Sourcegraph services: none
-  # Ports exposed to the public internet: 7080 (HTTP) and/or 7443 (HTTPS)
-  #
-  # Note: nginx/nginx/sourcegraph_backend.conf lists all of the sourcegraph-frontend
-  # network addresses that Nginx should proxy. nginx/nginx/sourcegraph_backend.conf
-  # needs to be updated with new addresses when scaling the number of soucegraph-frontend
-  # replicas.
-  nginx:
-    container_name: nginx
-    image: 'index.docker.io/library/nginx:1.17.7@sha256:8aa7f6a9585d908a63e5e418dc5d14ae7467d2e36e1ab4f0d8f9d059a3d071ce'
+  # Ports exposed to the public internet: 80 (HTTP) and 443 (HTTPS)
+  #
+  # Sourcegraph ships with a few builtin templates that cover some common HTTP/HTTPS configurations:
+  # - HTTP only (default)
+  # - HTTPS with Let's Encrypt (staging/production - strongly recommended that you test with the staging
```

Comment:
```suggestion
  # - HTTPS with Let's Encrypt (see caddy/builtins/https.lets-encrypt-staging.Caddyfile for details)
```

Reply:
@slimsag I think we need to explicitly call out to use the staging one first. Being banned from creating certs for a week is a real possibility.

Reply:
Agreed, I suggest doing this below and not here though.